updates/1/updates.sql

```diff
@@ -1 +1,24 @@
-FILL ME IN
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xaac41EC512808d64625576EDdd580e7Ea40ef8B2", 18, NULL);
+INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xaac41EC512808d64625576EDdd580e7Ea40ef8B2", "C", "Gameswap", "GSWAP", 11155251, NULL, "gameswap", "gswap", "0xaac41EC512808d64625576EDdd580e7Ea40ef8B2");
```

Comment:
```suggestion
INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xaac41EC512808d64625576EDdd580e7Ea40ef8B2", 18, NULL); INSERT INTO assets(identifier,type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xaac41EC512808d64625576EDdd580e7Ea40ef8B2", "C", "Gameswap", "GSWAP", 11155251, NULL, "gameswap", "gswap", "0xaac41EC512808d64625576EDdd580e7Ea40ef8B2");
*
```

A change concerning the same asset should have all its statements in the same line.

Additionally as per the readme each change should be comprised of two things:

1. Line 1: The change
2. Line 2: The addition statements if line 1 is not a simple addition. If it's a simple addition it should be an `*`.

Reply:
With the two lines I tought it was `INSERT INTO ethereum_tokens` + `INSERT INTO assets`

Reply:
no. For an example please look at the tests in rotki for the assets updates. I think this may make it much clearer: https://github.com/rotki/rotki/blob/develop/rotkehlchen/tests/api/test_assets_updates.py